dev/io.openliberty.jakartaee10.internal_fat/fat/src/io/openliberty/jakartaee10/internal/tests/JakartaEE10Test.java

```diff
@@ -75,24 +79,44 @@ private static Set<String> getCompatFeatures() {
         return compatFeatures;
     }
 
+    private static final String ALL_COMPAT_OL_FEATURES = "AllEE10CompatFeatures_OL_ONLY";
     private static final String ALL_COMPAT_FEATURES = "AllEE10CompatFeatures";
+
     @ClassRule
-    public static RepeatTests repeat = RepeatTests
-                    .with(new FeatureReplacementAction()
-                                    .removeFeature("webProfile-10.0")
-                                    .addFeature("jakartaee-10.0")
-                                    .withID("jakartaee10")
-                                    .fullFATOnly())
-                    .andWith(new FeatureReplacementAction()
-                                    .removeFeature("jakartaee-10.0")
-                                    .addFeature("webProfile-10.0")
-                                    .withID("webProfile10")
-                                    .fullFATOnly())
-                    .andWith(new FeatureReplacementAction()
-                                    .removeFeature("webProfile-10.0")
-                                    .removeFeature("jakartaee-10.0")
-                                    .addFeatures(getCompatFeatures())
-                                    .withID(ALL_COMPAT_FEATURES)); //LITE
+    public static RepeatTests repeat;
+
+    static {
+        Set<String> olCompatFeatures = getCompatFeatures(true);
+        Set<String> allCompatFeatures = getCompatFeatures(false);
```

Comment:
An enum rather than a boolean would make calls like this clearer.

Reply:
I will take that into consideration when I move the FeatureUtilities class into fattest.simplicity to avoid the dual maintenance.